routes/owner.js

```diff
@@ -2,62 +2,78 @@ const mongoose = require('mongoose')
 const express = require('express')
 const { Hotel, validate } = require('../models/hotel')
 const { Reservation } = require('../models/reservation')
+const ApiError = require('../helpers/apiError')
 const router = express.Router()
+const { isHotelOwner } = require('../middleware/role')
 
-// TODO: add auth middlewear, when it will be ready
+router.get('/hotels', isHotelOwner, async (req, res) => {
+  const hotels = await Hotel.find()
 
-router.get('/hotels', async (req, res) => {
-  const hotel = await Hotel.find()
-
-  res.status(200).send(hotel)
+  res.status(200).json({ hotels: hotels })
 })
 
-router.post('/hotel', async (req, res) => {
+router.post('/hotel', isHotelOwner, async (req, res) => {
   const { error } = validate(req.body)
-  if (error) return res.status(400).send(error.details[0].message)
+  if (error) throw new ApiError(400, error.details[0].message)
 
   const hotel = new Hotel(req.body)
 
   await hotel.save()
 
-  res.status(200).send(hotel)
+  res.status(200).json({ hotel: hotel })
```

Comment:
On front end:
Const hotel = res.hotels
Why we can't make it as simple as possible...
Just send(hotel) lol